modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -16,8 +11,10 @@ metadata:
     app.kubernetes.io/component: workspace-secret
 ...
 ----
-+
-Annotations must indicate the given secret is mounted as a file, provide the mount path, and, optionally, specify the name of the container in which the secret is mounted. If there is no target-container annotation, the secret will be mounted into all user containers of the {prod-short} workspace, but this is applicable *_only for the {prod-short} version 7.14_*.
+
+The following examples show more `metadata` of a secret:
```

Comment:
As a user, I want to know why this metadata is not in the first example. When should I add or remove them?

Reply:
```suggestion
The following examples show optional `metadata` you can use with secrets for a variety of reasons:
```